### PR TITLE
 past experiments on home page

### LINKED ISSUE #1951: Past experiments should be visible without addon installed
Looks like the 'view past experiments' button is gone from dev, so there's no way for new users to see the universal search graduation report.

We also need to make past experiments visible from the single experiment details page, so users can browse to all the experiments regardless of which page they are on.

Split off from #1945.

### PR DESCRIPTION
**Pull request by johngruen**
@dannycoates could you take a look since this is mostly refactoring your code?

Fixes #1951

### CODE REVIEW

File: frontend/test/app/components/PastExperiments-test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { expect } from 'chai';xe
```
Comment (pdehaan) [inline frontend/test/app/components/PastExperiments-test.js]: Looks like a bonus "xe" at the end there.

Comment (johngruen) [reply frontend/test/app/components/PastExperiments-test.js]: BONUS!

**Comment by johngruen** [comment]
Given the ongoing difficulties in #1945, we might not want to use this PR yet

**Comment by jaredhirsch** [comment]
@dannycoates sorry about the notification noise, I was trying to figure out how to get the reviewer avatar visible from the main PR list. I guess assignees, not reviewers, are the scannable thing

**Review by dannycoates — APPROVED** [review]
sure, why not 👍

**Comment by johngruen** [comment]
Oh shit

**Comment by johngruen** [comment]
I don;t think we should have merged this right now

**Comment by dannycoates** [comment]
> I don;t think we should have merged this right now

no regrets. keep pushing forward

**Comment by johngruen** [comment]
YOLO

**Pull request merged**